pkg/accelerator/accelResourcePool.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/types"
 )
 
+const (
+	accelPoolType = "accel"
```

Comment:
@adrianchiris what pool name do you prefer for this kind?

Reply:
accel is OK imo

Reply:
on a second thought.

we have: `net-sf`, `net-pci`  so lets have this one as `net-accel`  :)

Reply:
done